internal/cli/command/form/web/main.js

```diff
@@ -0,0 +1,2333 @@
+(window["webpackJsonp"] = window["webpackJsonp"] || []).push([["main"],{
```

Comment:
where does this file come from? do we really want to add it to git?

Reply:
We need to decide where should we keep the source code. I'd prefer to have it inside this repository.

Reply:
but this is generated code, isn't this?

Reply:
Yes, for now, it is.

Reply:
Added the client source as well.